src/javascripts/cookie-functions.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Cookie functions
+ * ================
+ *
+ * Used by the cookie banner component and cookies page pattern.
+ *
+ * Includes function `Cookie()` for getting, setting, and deleting cookies, and
+ * functions to manage the users' consent to cookies.
+ */
+
+/* Name of the cookie to save users cookie preferences to. */
+const CONSENT_COOKIE_NAME = 'cookies_policy'
```

Comment:
I don't think we can use `const`?

Reply:
Oh really? :( How come?

Reply:
I don't think we transpile these JS files. I think this PR passes because the code in this file isn't being called or imported anywhere. Once you do that ([example](https://github.com/alphagov/govuk-design-system/compare/ldeb-add-cookie-functions...add-gtm-snippet)), you get errors on build/start from UglifyJS:

```
Error: UglifyJS Error: {"message":"Unexpected token: keyword «const»","filename":"javascripts/application.js","line":9048,"col":1,"pos":309281}
```

Reply:
Oh no 😢 Is there alternative to `const` that is good to use?

Reply:
Looks like there is no way to transpile `const` anyway.

https://stackoverflow.com/questions/57912755/javascript-es5-transpile-for-es6-const#comment102245785_57912755

I will just substitute with `var`.

Reply:
Do you know if upgrading `uglify-js` allows us to use const? The release we are using is 2 years old now.

Reply:
Bumping `metalsmith-uglify` in main and rebasing this PR means we can use `const` now 🎉